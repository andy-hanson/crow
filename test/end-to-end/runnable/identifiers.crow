# This tests 'isAllowedUnicodeIdentifierChar'
main void()
	# Latin
	crow string = "crow"
	info log crow

	# Latin extended
	kráka string = "kráka"
	info log kráka

	# Greek and Coptic
	κοράκι string = "κοράκι"
	info log κοράκι

	# Cyrillic
	ворона string = "ворона"
	info log ворона

	# Hebrew
	עורב string = "עורב"
	info log עורב

	# Arabic
	غراب string = "غراب"
	info log غراب
	کلاغ string = "کلاغ"
	info log کلاغ
	کوا string = "کوا"
	info log کوا

	# Devanagari
	কাক string = "কাক"
	info log কাক
	कौआ string = "कौआ"
	info log कौआ
	कावळा string = "कावळा"
	info log कावळा

	# Bengali
	कऊआ string = "कऊआ"
	info log कऊआ

	# Gurmukhi
	ਕਾਂ string = "ਕਾਂ"
	info log ਕਾਂ

	# Gujarati
	કાગડો string = "કાગડો"
	info log કાગડો

	# Tamil
	காகம் string = "காகம்"
	info log காகம்

	# Telugu
	కాకి string = "కాకి"
	info log కాకి

	# Tibetan
	ཁྭ་ཏ་ནག་པོ། string = "ཁྭ་ཏ་ནག་པོ།"
	info log ཁྭ་ཏ་ནག་པོ།

	# Latin extended additional
	quạ string = "quạ"
	info log quạ

	# Hiragana
	からす string = "からす"
	info log からす

	# Katakana
	カラス string = "カラス"
	info log カラス

	# CJK Unified Ideographs
	乌鸦 string = "乌鸦"
	info log 乌鸦

	# Hangul
	까마귀 string = "까마귀"
	info log 까마귀
