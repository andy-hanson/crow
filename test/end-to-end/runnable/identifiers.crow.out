info: crow
info: kráka
info: κοράκι
info: ворона
info: עורב
info: غراب
info: کلاغ
info: کوا
info: কাক
info: कौआ
info: कावळा
info: कऊआ
info: ਕਾਂ
info: કાગડો
info: காகம்
info: కాకి
info: ཁྭ་ཏ་ནག་པོ།
info: quạ
info: からす
info: カラス
info: 乌鸦
info: 까마귀
